libs/ebpf_ext/ebpf_ext_tracelog.h

```diff
@@ -27,6 +27,7 @@ ebpf_ext_trace_terminate();
 #define EBPF_EXT_TRACELOG_KEYWORD_SOCK_ADDR 0x20
 #define EBPF_EXT_TRACELOG_KEYWORD_SOCK_OPS 0x40
 #define EBPF_EXT_TRACELOG_KEYWORD_PROCESS 0x80
+#define EBPF_EXT_TRACELOG_KEYWORD_PKTMON 0x81
 
```

Comment:
Up to now each of these tracelog keywords seemed to be a flag (power of 2) - I thought tracelog keywords were just integers, so I don't know why that was the case, but should this one be a power of 2 to match?  Or were those existing ones just weird?

Reply:
Please consider this branch is still under heavy development, not ready for reviews yet.